projects/storefrontstyles/scss/components/misc/search/_search.scss

```diff
@@ -0,0 +1,36 @@
+%cx-search {
```

Comment:
I cannot find any usages of `cx-search` component in our code base.
@Pucek9 wouldn't you mind if we remove the search component with its styles?